### DISCUSSION TITLE
x64 Work and Testing

### DISCUSSION THREAD

**Discussion by tadanokojin** [PCSX2/pcsx2#4102] [General]
https://github.com/PCSX2/pcsx2/pull/3451 is closed now. I want to keep track of discussion/testing about what remains for x64 and any new ideas (better register management, etc) borne of it.

[Windows Builds](https://github.com/PCSX2/pcsx2/actions?query=workflow%3A%22Windows+Build%22+branch%3Amaster)

So far I'm aware of:
- ~~folder memory cards are broken (addressed in https://github.com/PCSX2/pcsx2/pull/4113)~~
- ~~gsdx sw x64 is broken on Windows~~
- ~~gsdx sw x64 lacks mipmapping~~

**Comment by gregory38** [comment]
If I may, the first step is to make x64 work fine and then drop 32 bits.
Then feel free to rewrite everything to improve the recompiler speed.

**Reply by tadanokojin** [reply]
I agree. Focus should be on getting everything done.

**Reply by fuhuan416** [reply]
How does Vulkan feel :)

**Comment by tadanokojin** [comment]
The memory card stuff should be sorted now, so Linux at least is ready to go. For GSdx, I understand tellow has it mostly working and @GovanifY is planning to take a closer look. Will need to sort out mipmapping but after those two we shall be ready for public release (and any fun bugs that come).

**Comment by TellowKrinkle** [comment]
#3577 Now works on Windows and has mipmapping

**Comment by Mrlinkwii** [comment]
64 bit has been working for a good while  now , so i think this discussion  can be closed , well done :+1:
